test/e2e/tests/test_hpo.py

```diff
@@ -56,7 +56,7 @@ def xgboost_hpojob():
     yield (reference, resource)
 
     if k8s.get_resource_exists(reference):
-        _, deleted = k8s.delete_custom_resource(reference, 3, 10)
+        _, deleted = k8s.delete_custom_resource(reference, cfg.JOB_DELETE_WAIT_PERIODS, cfg.JOB_DELETE_WAIT_PERIODS)
```

Comment:
I think you meant to use cfg.JOB_DELETE_WAIT_LENGTH here
```suggestion
        _, deleted = k8s.delete_custom_resource(reference, cfg.JOB_DELETE_WAIT_PERIODS, cfg.JOB_DELETE_WAIT_LENGTHS)
```

Reply:
thx, done